Fail fast property for missing pod labels

GavrilovSV opened the pull request:
This is a possible solution to resolve #323

pgressa commented:
@GavrilovSV hey, thanks for contribution! I'll review your PR. Could I also ask you to extend the documentation? Thanks

pgressa commented:
@GavrilovSV I'm thinking the `fail-fast` is generic name of the property that can cause confusion on first sight as it's not evident that it is related to the `pod-labels`. Maybe `exception-on-pod-labels-missing` is more specific, or other ideas are welcomed. Feel free to inspire in Micronaut configuration reference https://docs.micronaut.io/latest/guide/configurationreference.html.

Otherwise I don't have a problem with the provided source code.

GavrilovSV commented:
@pgressa hey, thanks for the review! I agree that `fail-fast` may be too generic for this specific case, so I've renamed the property to `exception-on-pod-labels-missing`, as you've suggested, and updated the docs
